benchmarks/fs2/src/main/scala/laserdisc/fs2/parallel/baseline/BitVectorChannelCodec.scala

```diff
@@ -0,0 +1,40 @@
+package laserdisc
+package fs2
+package parallel
+package baseline
+
+import _root_.fs2.{Chunk, Pipe, Pull, Stream}
+import cats.MonadError
+import laserdisc.protocol._
+import scodec.bits.BitVector
+
+private[fs2] object BitVectorChannelCodec {
+  def send[F[_]: MonadError[*[_], Throwable]](socketChannel: Pipe[F, Byte, Unit]): Pipe[F, BitVector, Unit] =
+    _.flatMap(bits => Stream.chunk(Chunk.bytes(bits.toByteArray)))
+      .through(socketChannel)
```

Comment:
so something in here is fishy...

Reply:
my target now is `toByteArray`. We'll see.